tests/unit/amo/components/TestAddonReviewListItem.js

```diff
@@ -197,6 +198,14 @@ describe(__filename, () => {
     expect(root.find('.AddonReviewListItem-edit')).toHaveLength(0);
   });
 
+  it('assigns isOwner property if you wrote the review', () => {
```

Comment:
Might be useful to add an extra test ensuring the className is NOT output when the user isn't the owner of a review as well.